capz/run-capz-e2e.sh

```diff
@@ -143,9 +143,9 @@ create_cluster(){
             fi
         fi
         echo "Using $template"
-        
-        az capi create -mg "${CLUSTER_NAME}" -y -w -n "${CLUSTER_NAME}" -l "$AZURE_LOCATION" --template "$template" --tags creationTimestamp="$(date -u '+%Y-%m-%dT%H:%M:%SZ')"
-        
+
+        az capi create --debug -mg "${CLUSTER_NAME}" -y -w -n "${CLUSTER_NAME}" -l "$AZURE_LOCATION" --template "$template" --tags creationTimestamp="$(date -u '+%Y-%m-%dT%H:%M:%SZ')" --external-cloud-provider=false
```

Comment:
@mboersma - why are we setting external cloud provider to false?
Our templates expect external cloud provider 
https://github.com/kubernetes-sigs/windows-testing/blob/e0a358a80520a0c0b59810023a044f582b1ed619/capz/templates/windows-base.yaml#L77 and we build the required images if needed as part of this script.

Reply:
This was an addition from me trying to get the tests to pass. We already install it so didn't want to the extension to conflict with the install we do: https://github.com/kubernetes-sigs/windows-testing/blob/e0a358a80520a0c0b59810023a044f582b1ed619/capz/run-capz-e2e.sh#L204

Reply:
the version of cloud-provider that the extension knows about won't work in CI, IIRC